lucee-java/lucee-core/src/lucee/runtime/config/ConfigServerImpl.java

```diff
@@ -107,7 +107,7 @@ public final class ConfigServerImpl extends ConfigImpl implements ConfigServer {
      * @param configDir
      * @param configFile
      */
-    protected ConfigServerImpl(CFMLEngineImpl engine,Map<String,CFMLFactory> initContextes, Map<String,CFMLFactory> contextes, Resource configDir, Resource configFile) {
+	public ConfigServerImpl(CFMLEngineImpl engine, Map<String, CFMLFactory> initContextes, Map<String, CFMLFactory> contextes, Resource configDir, Resource configFile) {
```

Comment:
Did we make this `public` on purpose?

Reply:
I was going to attempt to write a test, and forgot to revert it